aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -34,6 +34,7 @@ pub enum FeatureFlag {
     StructConstructors,
     PeriodicalRewardRateReduction,
     PartialGovernanceVoting,
+    SignatureCheckerV2,
```

Comment:
Do you wish to enable this flag for 1.4? If so we probably will need to add the flag to the `release.yaml` that described the flags to enable.